package.json

```diff
@@ -11,7 +11,7 @@
     "start": "node $NODE_APP_OPTIONS app.js",
     "test:acceptance:_run": "mocha --recursive --reporter spec --timeout 15000 --exit $@ test/acceptance/js",
     "test:acceptance": "npm run test:acceptance:_run -- --grep=$MOCHA_GREP",
-    "test:unit:_run": "mocha --recursive --reporter spec --timeout 5000 $@ test/unit/js",
+    "test:unit:_run": "mocha --recursive --reporter spec $@ test/unit/js",
```

Comment:
That will not work -- cloudbuild will timeout #183. Adjusting it in the dev-env is not straight forward -- the entry is stored in a json file. I will revert it here and create a ticket to fix this properly.

Reply:
https://github.com/overleaf/issues/issues/3529